pkg/check/pushsync/check_concurrent.go

```diff
@@ -14,7 +14,7 @@ func CheckConcurrent(c *bee.Cluster, o Options) (err error) {
 	rnds := random.PseudoGenerators(o.Seed, o.UploadNodeCount)
 	fmt.Printf("Seed: %d\n", o.Seed)
 
-	ng := c.NodeGroup(o.NodeGroup)
+	ng := c.NodeGroup("skip")
```

Comment:
is this really the value?

Reply:
`NodeGroup` attribute has been removed from the options so I needed a string here, and using something meaningful like "bee" could have been misleading, but I will change it.